integration-tests/load/go.mod

```diff
@@ -27,6 +27,8 @@ require (
 	go.uber.org/ratelimit v0.3.0
 )
 
+require github.com/smartcontractkit/chainlink-protos/orchestrator v0.2.0 // indirect
```

Comment:
This is weird that it is on its own line separate from the rest

Reply:
It was a simple `go mod tidy`
While I could manually edit, I'd prefer not to unless there's a specific reason it might cause problems.